src/main/java/de/wejago/mqtt2influx/service/JsonSubscriber.java

```diff
@@ -65,7 +65,17 @@ private void addMapEntriesByExistingDeviceMappings(Map<String, String> deviceMap
         Map<String, String> messageValueMap = objectMapper.convertValue(entry.getValue(), Map.class);
```

Comment:
The problem is that the ObjectMapper sometimes parses the values properly as a Double and not as a String!
Then the test fails in line 69 because the result is a Double and not a String as expected!
You need to use `new TypeReference<Map<String, String>>(){}` instead of the `Map.class`. That way the ObjectMapper knows that the value MUST be a String.